src/modules/auth/login.controller.ts

```diff
@@ -132,4 +132,28 @@ export class LoginController {
 
     return res.send({ token, refreshToken });
   }
+
+  @Get('auth/status')
+  async status(@Req() req: Request) {
+    const accessTokenString =
+      req.cookies[this.configService.get<string>('JWT_ACCESS_TOKEN_NAME')] ||
```

Comment:
not a big deal but shouldn't header be primary source of truth?

Reply:
I think that depends from the environment, but token is first trying to be extracted from header in the login strategy.